cam0:
  T_cam_imu:
    [1.0, 0.0, 0.0, 0.0,
     0.0, 1.0, 0.0, 0.0,
     0.0, 0.0, 1.0, 0.0,
     0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
# k1, k2, t1, t2, k3
  distortion_coeffs: [-0.161357, 0.087973, -0.000643, -0.000262, 0.000000]
  distortion_model: plumb_bob
  intrinsics: [748.586631,  750.4137825,  505.0316095, 511.779532]
  resolution: [1024, 1024]
  rostopic: 17041930
cam1:
  T_cam_imu:
    [1.0, 0.0, 0.0, 0.0,
     0.0, 1.0, 0.0, 0.0,
     0.0, 0.0, 1.0, 0.0,
     0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.160887, 0.087520, -0.000100, -0.000319, 0.000000]
  distortion_model: plumb_bob
  intrinsics: [749.8574, 751.2651, 514.6226, 502.5214]
  resolution: [1024, 1024]
  rostopic: 17041931
cam2:
  T_cam_imu:
    [1.0, 0.0, 0.0, 0.0,
     0.0, 1.0, 0.0, 0.0,
     0.0, 0.0, 1.0, 0.0,
     0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.162011, 0.092032, -0.000002, -0.000388, 0.000000]
  distortion_model: plumb_bob
  intrinsics: [752.0638, 753.3976, 512.3292, 509.9695]
  resolution: [1024, 1024]
  rostopic: 17041932
cam3:
  T_cam_imu:
    [1.0, 0.0, 0.0, 0.0,
     0.0, 1.0, 0.0, 0.0,
     0.0, 0.0, 1.0, 0.0,
     0.0, 0.0, 0.0, 1.0]
  camera_model: pinhole
  distortion_coeffs: [-0.159304, 0.087633, 0.000369, -0.000861, 0.000000]
  distortion_model: plumb_bob
  intrinsics: [752.0853,  753.235,  513.7911,  510.5871]
  resolution: [1024, 1024]
  rostopic: 17041925

